src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -65,52 +67,272 @@ ucs_status_t uct_rdmacm_cm_reject(struct rdma_cm_id *id)
     return UCS_OK;
 }
 
-ucs_status_t uct_rdmacm_cm_get_cq(uct_rdmacm_cm_t *cm, struct ibv_context *verbs,
-                                  struct ibv_cq **cq_p)
+static ucs_status_t
```

Comment:
should return void. the query result is saved already to 'ctx'

Reply:
@yosefe Changed to uct_rdmacm_cm_device_context_init(), the return value is used to indicate the status of calling the function now.